lightkurve/lightcurve.py

```diff
@@ -291,16 +291,13 @@ def flatten(self, window_length=101, polyorder=2, return_trend=False,
         for l, h in zip(low, high):
             # Reduce `window_length` and `polyorder` for short segments;
             # this prevents `savgol_filter` from raising an exception
-            segment_window_length, segment_polyorder = window_length, polyorder
-            if segment_window_length > (h - l):
-                segment_window_length = h - l
-                if (segment_window_length % 2) == 0:
-                    segment_window_length -= 1  # window_length must be odd
-            if segment_polyorder >= segment_window_length:
-                segment_polyorder = segment_window_length - 1
-            trend_signal[l:h] = signal.savgol_filter(x=lc_clean.flux[l:h],
-                                                     window_length=segment_window_length,
-                                                     polyorder=segment_polyorder, **kwargs)
+            # If the segment is too short, just take the median
+            if np.any([window_length > (h - l), (h - l) < break_tolerance]):
```

Comment:
I wonder if it is more clear to write: `if (window_length > (h - l)) or ((h - l) < break_tolerance):`